incubator/hnc/internal/reconcilers/object.go

```diff
@@ -53,6 +54,9 @@ const (
 	actionNop     syncAction = "no-op"
 
 	unknownSourceNamespace = "<unknown-source-namespace>"
+
+	selectorAnnot     = "propagate.hnc.x-k8s.io/select"
+	treeSelectorAnnot = "propagate.hnc.x-k8s.io/treeSelect"
```

Comment:
These constants should be in the API package. Also, you shouldn't be repeating yourself so much - "hnc.x-k8s.io" is already called something like `api.MetaGroup`, and you should have a constant for the `propagate.` part as well.

Reply:
https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/api/v1alpha2/hierarchy_types.go#L32

Reply:
done